include/osquery/status.h

```diff
@@ -69,7 +77,9 @@ class Status {
    * success or failure of an operation. On successful operations, the idiom
    * is for the message to be "OK"
    */
-  std::string getMessage() const { return message_; }
+  std::string getMessage() const {
```

Comment:
Some times we need just to see the message without storing it somewhere. From that point of view here and in `toString()` and in `what()` is better to use just a const ref to string as return value, is'n it?

Reply:
it's clan format modification, no logic changes here